src/common/contracts/core/campaigns/client.ts

```
@@ -49,23 +49,23 @@
 }: {
   args: CampaignInputs & { campaign_id: CampaignId };
 }) => {
-  return contractApi.call("update_campaign", {
+  return contractApi.call<{}, Campaign>("update_campaign", {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)